tests/libmonitoring/prometheus.go

```diff
@@ -82,6 +82,16 @@ func WaitForMetricValueWithLabels(client kubecli.KubevirtClient, metric string,
 	}, 3*time.Minute, 1*time.Second).Should(BeNumerically("==", expectedValue))
 }
 
+func WaitForMetricValueWithLabelsToBe(client kubecli.KubevirtClient, metric string, labels map[string]string, offset int, comparator string, expectedValue float64) {
+	EventuallyWithOffset(offset, func() float64 {
+		i, err := GetMetricValueWithLabels(client, metric, labels)
+		if err != nil {
+			return -1
+		}
+		return i
+	}, 3*time.Minute, 1*time.Second).Should(BeNumerically(comparator, expectedValue))
```

Comment:
@machadovilaca small question: why 3 minutes?

Reply:
same value we used before, basically:

- 1min for controllers updates
- Prometheus scrapes more or less every 30s, so +1min minimum in case it just scrapped
- +1min for margin

not too low, not unnecessarily high

Reply:
@machadovilaca thanks!